### PR TITLE
feat(titus): default container attrs resolver

### PR DESCRIPTION
**Pull request by emjburns**
The titus team recently started setting some default container attributes. This adds them to a user spec if they're not present.

Default values are pulled from the config.

You can turn it off via fast property in case we mess something up :)

### CODE REVIEW

File: keel-titus-plugin/src/test/kotlin/com/netflix/spinnaker/keel/titus/ContainerAttributesResolverTests.kt [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      test("account and subnet are set") {
+        validateKeysSet(resolved)
+      }
+    }
+
+    context("account is set") {
+      fixture {
+        Fixture(
+          ContainerAttributesResolver(defaults, clouddriverService, springEnv),
+          baseSpec.copy(_defaults = baseSpec.defaults.copy(containerAttributes = mapOf(accountKey to awsAccountId)))
```
Comment (lorin) [inline keel-titus-plugin/src/test/kotlin/com/netflix/spinnaker/keel/titus/ContainerAttributesResolverTests.kt:90]: I would recommend using a different account id here (or adding another test with a different account id), to catch the bug where the user specifies a custom account id, and the resolver clobbers it with the default.

**Review by robfletcher — APPROVED** [review]

**Review by lorin — APPROVED** [review]
Small comment on the tests, otherwise LGTM!

File: keel-titus-plugin/src/test/kotlin/com/netflix/spinnaker/keel/titus/ContainerAttributesResolverTests.kt (hunk 1 above)
Comment (emjburns) [reply keel-titus-plugin/src/test/kotlin/com/netflix/spinnaker/keel/titus/ContainerAttributesResolverTests.kt:90]: ok i'll do that, good point

Comment (emjburns) [reply keel-titus-plugin/src/test/kotlin/com/netflix/spinnaker/keel/titus/ContainerAttributesResolverTests.kt:90]: oh wait i do this in the following test!

**Pull request merged**